frontend/src/app/components/crm/leads/leads.component.ts

```diff
@@ -12,9 +12,11 @@ import { AfterViewInit, Component, OnInit, ViewChild } from '@angular/core';
 import { MatPaginator } from '@angular/material/paginator';
 import { MatSort } from '@angular/material/sort';
 import { MatTableDataSource } from '@angular/material/table';
-import { SelectionModel } from '@angular/cdk/collections';
+
 import { MatDialog } from '@angular/material/dialog';
 
+import {SelectionModel} from '@angular/cdk/collections';
```

Comment:
Add space before and after SelectionModel

Reply:
alright